app/helpers/restaurants_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module RestaurantsHelper
+end
```

Comment:
I would suggest to not commit empty helpers

Reply:
I deleted empty helper files